server/resourcemanager/testutil/test_resource.go

```diff
@@ -81,53 +41,62 @@ func TestResourceTypeOperations(t *testing.T, rt ResourceTypeTest, operations []
 	t.Helper()
 
 	t.Run(rt.ResourceType, func(t *testing.T) {
-
 		for _, op := range operations {
-			t.Run(string(op.name()), func(t *testing.T) {
+			t.Run(string(op.name), func(t *testing.T) {
 				op := op
 				t.Parallel()
 
 				testOperation(t, op, rt)
 			})
 		}
-
 	})
 }
 
 func testOperation(t *testing.T, op operationTester, rt ResourceTypeTest) {
 	t.Helper()
 
-	for _, ct := range contentTypes {
+	for _, ct := range contentTypeConverters {
 		t.Run(ct.name, func(t *testing.T) {
 			ct := ct
 			t.Parallel()
 
-			testContentType(t, op, ct, rt)
+			testOperationForContentType(t, op, ct, rt)
 		})
 	}
 }
 
-func testContentType(t *testing.T, op operationTester, ct contentType, rt ResourceTypeTest) {
+func testOperationForContentType(t *testing.T, op operationTester, ct contentTypeConverter, rt ResourceTypeTest) {
 	t.Helper()
 
 	router := mux.NewRouter()
 	testServer := httptest.NewServer(router)
 	testBridge := rt.RegisterManagerFn(router)
 
 	if rt.Prepare != nil {
-		rt.Prepare(t, op.name(), testBridge)
+		rt.Prepare(t, op.name, testBridge)
 	}
 
-	req := op.buildRequest(t, testServer, ct, rt)
-	resp := doRequest(t, req, ct, testServer)
+	// minor hack to pass "sortFields" to each operation
+	// without needing to propagate sortableHandler for each one
+	sortableHandler := testBridge.(rm.SortableHandler)
```

Comment:
Let people know what is missing

```suggestion
	sortableHandler, ok := testBridge.(rm.SortableHandler)
	if !ok {
		panic("provided testBridge does not implement `resourcemanager.SortableHandler` interface")
	}
```

Reply:
makes sense! I'll open another PR to add that